app/src/main/java/org/android/go/sopt/presentation/gallery/GalleryFragment.kt

```diff
@@ -0,0 +1,32 @@
+package org.android.go.sopt.presentation.gallery
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import org.android.go.sopt.databinding.FragmentGalleryBinding
+
+class GalleryFragment : Fragment() {
+    private var _binding: FragmentGalleryBinding? = null
+    private val binding: FragmentGalleryBinding
+        get() = requireNotNull(_binding!!) { "${this::class.java.simpleName}에서 에러가 발생했습니다." }
```

Comment:
```suggestion
        get() = requireNotNull(_binding) { "${this::class.java.simpleName}에서 에러가 발생했습니다." }
```

바인딩 변수가 null인 경우에 `requireNotNull`이 실행되지 않고 `!!`에 의해 NPE가 먼저 발생할 것 같습니다!